vispy/visuals/filters/clipping_planes.py

```diff
@@ -24,14 +24,15 @@ class PlanesClipper(Filter):
 
     VERT_CODE = """
     void clip() {
-        // Transform back to visual coordinates and clip based on that
-        $v_distance_from_clip = $clip_with_planes($itransform(gl_Position).xyz);
+        // pass position as varying for interpolation
+        $v_position = gl_Position;
```

Comment:
May be divide by `w` for good measure.
```suggestion
        $v_position = gl_Position / gl_Position.w;
```

Reply:
Uhm, this works without this changed and breaks with it. Check out the clipping planes example and change the fov... However, I'm not sure why that's the case, I also thought this should be divided by `w`.

Reply:
Ah, I think that when a value is interpolated (for the fragment shader), you want to use the "undivided" version.

Reply:
but then I would expect that I would need to divide it after interpolating... why is that not the case? :P

Reply:
What happens if you do? Do you see a difference?

Reply:
I see no difference, both with the clipping planes example and your code above. Changing fov looks perfectly fine with or without dividing by w.